articles/azure/azs-how-create-S2S-VPN.md

```diff
@@ -0,0 +1,222 @@
+---
+title: How to create a Site-to-Site connection in Azure Stack | UKCloud Ltd
+description: Create a S2S VPN gateway connection from an on-premises network to UKCLoud for Microsoft Azure 
+services: azure-stack
+author: Bailey Lawson
+toc_rootlink: Users
+toc_sub1: How To
+toc_sub2:
+toc_sub3:
+toc_sub4:
+toc_title: Create a Site-to-Site connection in Azure Stack
+toc_fullpath: Users/How To/azs-how-create-S2S-VPN.md
+toc_mdlink: azs-how-create-S2S-VPN.md
+---
+
+# How to create a Site-to-Site connection in Azure Stack
+
+## Overview
+
+This guide shows you how to use the UKCloud Azure Stack portal to create a Site-to-Site VPN from your on-premises network to your UKCloud for Microsoft Azure environment. 
+
+### Intended audience
+
+To complete the steps in this guide, you must have appropriate access to a subscription in the UKCloud Azure Stack portal.
+
+## Create a virtual network
+
+First, a virtual network will need to be created. This virtual network will be able to send and receive traffic through the virtual network gateway. A virtual network can not be associated with more than one gateway.

```

Comment:
Use active, rather than passive voice:
First, you'll need to create a virtual network.
You cannot associate a virtual network with more than one gateway.

Reply:
Fixed